config.toml.example

```diff
@@ -25,3 +25,20 @@ token = "<token>"
 [github.<another_orgname>]
 token = "<another_token>"
 
+[ldap]
+# This section is required by `qcontract-reconcile ldap-users`
+
+# (mandatory) ldap endpoint
+server = "ldap://ldap.example.com"
+# (optional) user search point
+base_dn = "ou=users,dc=example,dc=com"
+
+# This section is required by `qcontract-reconcile ldap-users`
+[gitlab]
+# These parameters can be overridden at project level
+server = "https://gitlab.example.com"
+token = "..."
+
+# One sub-section per managed GitLab project
+[gitlab.<prject_name>]
+project_id = "12345"
```

Comment:
OK, instead of `[gitlab]` and `[gitlab.<project_name>]` I suggest we have a single:

```
[app-interface-repo]
server = ...
token = ...
project_id = ...
```